modules/evm-bridge/src/lib.rs

```diff
@@ -250,6 +257,77 @@ impl<T: Config> EVMBridgeTrait<AccountIdOf<T>, BalanceOf<T>> for EVMBridge<T> {
 	}
 }
 
+pub struct LiquidationEvmBridge<T>(sp_std::marker::PhantomData<T>);
+
+impl<T: Config> LiquidationEvmBridgeT for LiquidationEvmBridge<T> {
+	fn liquidate(
+		context: InvokeContext,
+		collateral: EvmAddress,
+		repay_dest: EvmAddress,
+		amount: Balance,
+		min_repayment: Balance,
+	) -> DispatchResult {
+		// liquidation contract method hash
+		let mut input = Into::<u32>::into(Action::Liquidate).to_be_bytes().to_vec();
+
+		// append collateral ERC20 address
+		input.extend_from_slice(H256::from(collateral).as_bytes());
+		// append repay dest address
+		input.extend_from_slice(H256::from(repay_dest).as_bytes());
+		// append collateral amount
+		input.extend_from_slice(H256::from_uint(&U256::from(amount)).as_bytes());
+		// append minimum repayment amount
+		input.extend_from_slice(H256::from_uint(&U256::from(min_repayment)).as_bytes());
+
+		let info = T::EVM::execute(
+			context,
+			input,
+			Default::default(),
+			liquidation::LIQUIDATE.gas,
+			liquidation::LIQUIDATE.storage,
+			ExecutionMode::Execute,
+		)?;
+
+		Pallet::<T>::handle_exit_reason(info.exit_reason)
+	}
+
+	fn on_collateral_transfer(context: InvokeContext, collateral: EvmAddress, amount: Balance) {
+		// liquidation contract method hash
+		let mut input = Into::<u32>::into(Action::OnCollateralTransfer).to_be_bytes().to_vec();
+		// append collateral ERC20 address
+		input.extend_from_slice(H256::from(collateral).as_bytes());
+		// append collateral amount
+		input.extend_from_slice(H256::from_uint(&U256::from(amount)).as_bytes());
+
+		let _ = T::EVM::execute(
```

Comment:
is it OK to ignore the success status here?

Reply:
`on_collateral_transfer` is for notifying contracts, there isn't much could be done for runtime if something is wrong, other than probably logging the result

Reply:
So basically we assume if something goes wrong in that function it's the contract's fault and we are not responsible for its errors.